docs/2.6.0/versions.json

```diff
@@ -1,5 +1,5 @@
 {
   "daml": "2.6.0-snapshot.20230222.11475.0.ec147a82",
   "canton": "20230222",
-  "daml_finance": "1.1.3"
+  "daml_finance": "1.1.4"
```

Comment:
Do we want to change anything on the 2.6.0 Daml Finance documentation? 
If that is stable, maybe it should stay on 1.1.3?

Reply:
The end goal is to enable hoogle indexing, ideally as part of the `2.6` release.

What this change does is ensure that the daml finance doc is sourced from the docs repo and not from the assembly that we ship (hence the changes in the `toc`).

The `1.1.4` assembly also ships a hoogle index, but it will require an update in order to fix the hoogle base url, which is currently pointing to the wrong location.